src/sentry/tasks/integrations/slack/post_message.py

```diff
@@ -17,10 +17,10 @@
     max_retries=0,
 )
 def post_message(
+    integration_id: int,
```

Comment:
The change in `arg` order here doesn't matter as this is a task always called with:

```python
post_message.apply_async(
  kwargs={
    "integration_id": integration.id,
    "payload": payload,
    ...
  }
)
```